StarcounterSamples/Products
# Feature/19 add shops box

To test:

1. Download people app using warehouse
1. Using People add your main organization and shops (don't forget about the input field at bottom of the form)
1. Using Products/Settings select your main organization
1. Using Products/Organizations mark your shops -> currently you can't change parent organization using People
1. You should also probably create some products (using products app)

Known bugs:
- [ ] stock value for individual shops don't update in UI (client-side model is up-to-date) probably due to a bug in juicy-table-repeat
